api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -159,5 +158,41 @@
 			<column name="changed_by" type="int" />
 		</createTable>
 	</changeSet>
-    
+	<changeSet id="20181402-TRUNK-5339" author="esirkings">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="encounter_diagnosis" />
+			</not>
+		</preConditions>
+		
+		<comment>Creating encounter_diagnosis table</comment>
+		
+		<createTable tableName="encounter_diagnosis">
+			<column name="diagnosis_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="encounter_id" type="int"><constraints nullable="false" /></column>
+			<column name="condition_id" type="int"><constraints nullable="false" /></column>
+			<column name="certainty" type="varchar(255)"><constraints nullable="false" /></column>
+			<column name="rank" type="int"><constraints nullable="false" /></column>
+			<column name="uuid" type="char(38)"><constraints nullable="false" unique="true" /></column>
+			<column name="creator" type="int"><constraints nullable="false"/></column>
+			<column name="date_created" type="datetime"><constraints nullable="false"/></column>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="datetime" />
+			<column name="voided" type="boolean" defaultValueBoolean="false">
+				<constraints nullable="false" />
+			</column>
+			<column name="voided_by" type="int" />
+			<column name="date_voided" type="datetime" />
+			<column name="void_reason" type="varchar(255)"/>
+		</createTable>
+		
+		<addForeignKeyConstraint constraintName="encounter_diagnosis_encounter_id_fk"
```

Comment:
Can you also add foreign key constraints for the creator and chnaged_by

Reply:
Yeah sure